src/Sentry/User.cs

```diff
@@ -42,7 +42,7 @@ public sealed class User : IJsonSerializable
         get => _username;
         set
         {
-            if (_username != value)
+            if (_username is not value)
```

Comment:
Oh I read this as `null` checks somehow.

So we're checking `==` operator on the value we have stored + value set to spare the call to the native layer?

Reply:
yes
I'll roll back